input/pagecontent/general-requirements.md

```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
-      "resourceType" : "Patient",
-           ...
-           "_birthDate" : [
-               {
-                "extension" : [
-                  {
-                    "url" : "http://hl7.org/fhir/StructureDefinition/data-absent-reason",
-                    "valueCode" : "unknown"
-                  }
-                ]
-              }
-           ],
-          ...
-         }
-    ~~~
-
-1. For *coded* data elements:
-   - *example*, *preferred*, or *extensible* binding strengths (CodeableConcept, or Coding datatypes):
-      - if the source system has text but no coded data, only the text element is used.
-          - for Coding datatypes, the text only data is represented as a `display` element.
-      - if there is neither text or coded data:
-        - the appropriate "unknown" concept code **SHALL** be present if the binding strength is *extensible*
-        - if the value set does not have an appropriate "unknown" concept code, use `unknown` from the [DataAbsentReason Code System](http://terminology.hl7.org/CodeSystem/data-absent-reason).
-
-        Example: AllergyIntolerance resource where the manifestation is unknown.
-        ~~~
-        ...
-        "reaction" : [
-          {
-            "manifestation" : [
-              {
-                "coding" : [
-                  {
-                    "system" : "http://terminology.hl7.org/CodeSystem/data-absent-reason",
-                    "code" : "unknown",
-                    "display" : "unknown"
-                  }
-                ]
-              }
-            ]
-          }
-        ]
-        ...
-        ~~~
-
-   - *required* binding strength (CodeableConcept or code datatypes):
-      - the appropriate "unknown" concept code **SHALL** be present if available.
-      - if the value set does not have the appropriate “unknown” concept code you must use a concept from the value set otherwise the instance will not be conformant.
-
-        - For AU Core profiles, the following mandatory or conditionally mandatory* status elements with required binding have no appropriate "unknown" concept code:
-          - `AllergyIntolerance.clinicalStatus`*
-          - `Condition.clinicalStatus`*
-          - `Immunization.status`
-
-
-        *The clinicalStatus element is conditionally mandatory based on resource specific constraints. 
-
-
-### Suppressed Data
-In some circumstances, specific pieces of data may be hidden due to security or privacy reasons. 
+_Profile + Interaction Support_ refers to a system that represents digital health information using the content models of AU Core profiles AND implement the REST API interactions defined by the applicable AU Core capability statement.
 
-For elements with a minimum cardinality = 0 (including elements labelled *Must Support*), the element **SHALL** be omitted from the resource if they are suppressed.
+An [AU Core Responder](ActorDefinition-au-core-actor-responder.html) that declares 'Profile + Interaction Support' conformance:
```

Comment:
Different formatting here: 'Profile + Interaction Support' as opposed to using italics in 2.1.2.3 Profile Only Support:
"A system that [declares conformance](https://github.com/hl7au/au-fhir-core/pull/168/declaring-conformance.html#profile-only-support) as _Profile Only Support_ to an AU Core profile:"

Apply consistent formatting. 

There is one more instance of 'Profile + Interaction Support' below.

Reply:
All instances of "Profile + Interaction Support" fixed to be "Profile Support + Interaction Support".
Removed formatting of either italics or single comma.

Also fixed the wording that had referred to these as 'conformance tiers'. They are not; no such concept is defined.